runelite-client/src/main/java/net/runelite/client/plugins/nateplugins/skilling/nateminer/nateminer/MiningScript.java

```diff
@@ -18,15 +19,34 @@ public boolean run(MiningConfig config) {
 
             try {
                 if (Microbot.isMoving() || Microbot.isAnimating() || Microbot.pauseAllScripts) return;
-                if (Inventory.isFull()) {
-                    if (config.hasPickaxeInventory()) {
-                        Inventory.dropAllStartingFrom(1);
-                    } else {
-                        Inventory.dropAll();
+
+                /**
+                 *  Break handler logic:
+                 *      First check to see if the Run Time Timer has finished running
+                 *      Then notify that a break can be started
+                 *      Finally lets wait until the break is over.
+                 */
+                if (BreakHandlerScript.getHasRunTimeTimerFinished()) {
```

Comment:
Can this logic live inside the breakHandlerScript? I just want to avoid editing 50 scripts because something inside this code block has changed